File: ru.kmorozov.gbd.core/src/main/java/ru/kmorozov/gbd/core/logic/extractors/base/IImageExtractor.java
package ru.kmorozov.gbd.core.logic.extractors.base;

import ru.kmorozov.gbd.core.logic.Proxy.HttpHostExt;

/**
 * Created by sbt-morozov-kv on 15.11.2016.
 */
public interface IImageExtractor extends Runnable {

    void newProxyEvent(HttpHostExt proxy);
}

File: ru.kmorozov.gbd.core/src/main/java/ru/kmorozov/gbd/core/logic/extractors/base/IImageExtractor.kt
package ru.kmorozov.gbd.core.logic.extractors.base

import ru.kmorozov.gbd.core.logic.proxy.HttpHostExt

/**
 * Created by sbt-morozov-kv on 15.11.2016.
 */
interface IImageExtractor : Runnable {

    fun newProxyEvent(proxy: HttpHostExt)
}
